008.md

```diff
@@ -0,0 +1,254 @@
+# Binary Field Encodings
+
+Author: Anders Rune Jensen
```

Comment:
I think the original author was mix actually

Reply:
Okay, I can put both. I put you here because [the license text had so](https://github.com/ssbc/ssb-bfe-spec/blob/569653af880e8beb9e6c1d8fb0c80113492022f4/README.md?plain=1#L2)